Package/PluginInstaller.cs

```diff
@@ -192,6 +193,10 @@ IEnumerable<string> possiblePaths(string file)
                 }
                 catch (Exception e)
                 {
+                    if (step.FailOnNoExe && e is Win32Exception)
```

Comment:
It looks like you flipped the condition, and the action will still fail even if the condition is met. 

We probably want something like

```csharp
catch (Win32Exception) when (step.FailOnNoExe == false)
{
    log.Warning($"'{step.ExeFile}' not found, skipping action.");
}
```